.gitignore

```diff
@@ -19,3 +19,13 @@ pytest_session.txt
 .cache/
 .pytest_cache
 .coverage
+
+# Python
+**/*.egg-info
+
+# VSCode
+**/.vscode
```

Comment:
Technically you can/should put this in a global gitignore file. 
https://gist.github.com/subfuzion/db7f57fff2fb6998a16c Here's a link to help you set it up.

Reply:
Nice, didn't know about this. I'll make the change :+1